src/components/ChatEntry.js

```diff
@@ -1,22 +1,47 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
 
 const ChatEntry = (props) => {
+
+  const onLikeButtonClick = () => {
+    const updatedChatEntry = {
+      id: props.id,
+      sender: props.sender,
+      body: props.body,
+      timeStamp: props.timeStamp,
+      liked: !props.liked
+    };
+    props.onLiked(updatedChatEntry);
+  };
+  
+  const like = props.liked ? '❤️' : '🤍';
+
   return (
     <div className="chat-entry local">
```

Comment:
All of the chat entries display in one column rather than two because the second class value here is always the same, rather than being set to `local` for one user and `remote` for the other (see line 75 in ChatEntry.css for the relevant css).  One strategy is to use the props.sender in a ternary, but the syntax is messy:
```suggestion
    <div className={'chat-entry ' + (props.sender === 'Vladimir'? 'local' : 'remote')}>
```
Another strategy is to create a function that returns the correct classes, and call it here.